bindings/core/src/method_handler/client.rs

```diff
@@ -173,7 +173,21 @@ pub(crate) async fn call_client_method_internal(client: &Client, method: ClientM
         ClientMethod::GetHealth { url } => Response::Bool(client.get_health(&url).await?),
         ClientMethod::GetNodeInfo { url, auth } => Response::NodeInfo(Client::get_node_info(&url, auth).await?),
         ClientMethod::GetInfo => Response::Info(client.get_info().await?),
-        ClientMethod::GetPeers => Response::Peers(client.get_peers().await?),
+        ClientMethod::GetAccountCongestion { address } => {
+            // TODO: .as_account() can panic, accept account id or bech32 address?
```

Comment:
@thibault-martinez would you also accept the account id from the bindings side? Or should I just make the conversion return an error if a wrong address is provided?

Reply:
I would personally take IDs